compute_tools/src/config.rs

```diff
@@ -198,7 +198,7 @@ pub fn write_postgres_conf(
                 // Typically, this should be unreacheable,
                 // because we always set at least some shared_preload_libraries in the spec
                 // but let's handle it explicitly anyway.
-                writeln!(file, "shared_preload_libraries='neon,pgaudit'")?;
+                writeln!(file, "shared_preload_libraries='neon,anon,pgaudit'")?;
```

Comment:
I know that's a draft just to test that things work, but we mustn't preload `anon` to everyone by default. It should be optional and controlled by user. There is a new mechanism for preloading optional extension, work is tracked here https://github.com/neondatabase/cloud/issues/24412

The list of such extensions is here https://github.com/neondatabase/cloud/blob/026e0afff20bdbb864421fdd93ae78675a18ca38/goapp/controlplane/internal/pkg/compute/computespec/preload_libraries.go#L23